#Expyriment 1.0.0 (Python 3.11.9), .xpe-file, coding: cp1252
#date: Fri Sep 19 2025 00:19:32
#sha1: 140c45
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Windows', node='Madhusree', release='10', version='10.0.26100', machine='AMD64')
Time,Type,Event,Value,Detail,Detail2
410,stderr,received,'Exception ignored in: <function Visual.__del__ at 0x000001FADB6DE3E0>'
411,stderr,received,'Traceback (most recent call last):'
411,stderr,received,'  File "C:\\Users\\aviba\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\expyriment\\stimuli\\_visual.py", line 266, in __del__'
412,stderr,received,'    if self._compression_filename is not None:'
412,stderr,received,'       ^^^^^^^^^^^^^^^^^^^^^^^^^^'
412,stderr,received,"AttributeError: 'Line' object has no attribute '_compression_filename'"
412,stderr,received,'Traceback (most recent call last):'
412,stderr,received,'  File "C:\\Users\\aviba\\downloads\\cogsup-core1\\programming\\assignments\\week-2\\exercises\\labeled_shapes.py", line 40, in <module>'
413,stderr,received,'    triangle_line = stimuli.Line('
413,stderr,received,'                    ^^^^^^^^^^^^^'
413,stderr,received,"TypeError: Line.__init__() got an unexpected keyword argument 'start'"
